tests/models/test_tpu.py

```diff
@@ -0,0 +1,122 @@
+import os
+from unittest.mock import patch
+
+import pytest
+from pytorch_lightning import Trainer
+from pytorch_lightning.utilities.exceptions import MisconfigurationException
+from tests.base import EvalModelTemplate
+
+try:
+    import torch_xla
```

Comment:
is it enough just to test import? maybe rather try if a TPU is presented

Reply:
Will add this check :]

Reply:
@Borda I might require some help from the xla team here. I added a check for TPU but the tests get aborted due to some threading issue: notebook: https://www.kaggle.com/lezwon/pytorch-lightning-tpu-tests

Reply:
I think at least put there a todo note :]

Reply:
Sure. I will add a TODO note and check only for XLA import for now. Will that be fine?

Reply:
Sure, add i link to xla issue if created any.. And let's finish it together with CI deployment on TPU machine